lms/templates/financial-assistance/financial_assessment_form.underscore

```diff
@@ -31,7 +31,7 @@
 			<div class="data"><%- name %></div>
 		</div>
 		<div class="info-column">
-			<div class="title"><%- gettext('Country of residence') %></div>
+			<div id="user-country-title" class="title"><%- gettext('Country of residence') %></div>
```

Comment:
That's why I chose an ID.  It is a unique attribute.  There will be only one `user-country-title` on the page.  I wanted to communicate that by using an ID